### PR TITLE
Update mirroring-in-win32.md

### PR DESCRIPTION
**Pull request by DJm00n**
Fix code examples

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [globalization/localizability/mirroring-in-win32.md](https://github.com/MicrosoftDocs/globalization/blob/6b59fef983c729973a9f8b212f143f0150aa4451/globalization%2Flocalizability%2Fmirroring-in-win32.md) | 80 | 100 | 92 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0b02fd5a-5599-4ce5-8bea-c41806f9fe27) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: globalization/localizability/mirroring-in-win32.md [hunk 1]
```diff
@@ -350,10 +348,8 @@ HICON CreateMirroredIcon(HICON hiconOrg)
                     hbmMask = CreateBitmap(bm.bmWidth, bm.bmHeight,1, 1, NULL);
                     hbmOld = (HBITMAP)SelectObject(hdcBitmap, hbm);
                     hbmOldMask = (HBITMAP)SelectObject(hdcMask,hbmMask);
-                    DrawIconEx(hdcBitmap, 0, 0, hiconOrg,bm.bmWidth, _
-                    bm.bmHeight, 0, NULL, DI_IMAGE);
-                    DrawIconEx(hdcMask, 0, 0, hiconOrg, _
-                    bm.bmWidth,bm.bmHeight, 0, NULL, DI_MASK);
+                    DrawIconEx(hdcBitmap, 0, 0, hiconOrg,bm.bmWidth, bm.bmHeight, 0, NULL, DI_IMAGE);
```
Comment (jowilco) [inline globalization/localizability/mirroring-in-win32.md:351]: If you're going to improve the layout anyway, please could you add a space after the comma in "hiconOrg,bm.bmWidth", then on the next line "bm.bmWidth,bm.bmHeight"

**Review by jowilco — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [globalization/localizability/mirroring-in-win32.md](https://github.com/MicrosoftDocs/globalization/blob/a2ffd2c74f70f2ee23086951bd282a145cd6cbc7/globalization%2Flocalizability%2Fmirroring-in-win32.md) | 80 | 100 | 92 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1447944e-3fa0-46cd-8c7d-e8ce641e9028) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [globalization/localizability/mirroring-in-win32.md](https://github.com/MicrosoftDocs/globalization/blob/cb35b6d6b05f4d13c7406e4f60e88b94b6a114b8/globalization%2Flocalizability%2Fmirroring-in-win32.md) | 80 | 100 | 92 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/840b00fa-7e6d-43e7-a741-266bf9762663) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Review by jowilco — APPROVED** [review]

**Pull request merged**